fineract-provider/src/main/java/org/apache/fineract/portfolio/note/service/NoteWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -48,6 +50,8 @@
 @Slf4j
 public class NoteWritePlatformServiceJpaRepositoryImpl implements NoteWritePlatformService {
 
+    private static final String NOTE = "note";
```

Comment:
I'm very much against this, in such trivial cases. Especially if you are writing `"note"` just in the same file at line 84.

Reply:
Noted. It has been removed.

Reply:
I've found several (> 10) static (or not static :) ) "note" parameter definition, everywhere in Fineract. Something like this:
public static final String NOTE_PARAM = "note";
If you refactor the notes anyway, it would be nice to add this to a central place only once and everywhere (all the account types) could use this.
Do you think it fits this PR?

Reply:
Actually, static definitions of "note" appear in many places across the codebase. You can use the following command to locate them:

```bash
grep -rnw --include="*.java" '.' -e '[A-Za-z_][A-Za-z0-9_]*\s*=\s*"note"'
```

However, addressing this is something that can be tackled later. For now, I need to focus on other tasks related to implementing a type-safe API.